Core/OxidEeEvents.php

```diff
@@ -290,7 +141,25 @@ public static function createOrderTransactionTable()
             PRIMARY KEY (`TRANSACTIONNUMBER`)
         ) Engine=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci";
 
-        self::$oDb->Execute($sQuery);
+        self::$oDb->execute($sQuery);
+    }
+
+    /**
+     * Creates the module's payment method meta data table
+     *
+     * @since 1.2.0
+     */
+    public static function createPaymentMethodMetaDataTable()
```

Comment:
Also here i would name it like createPaymentMethodSettingsDataTable()

Reply:
See [above](#discussion_r289730250).